source/agora/consensus/data/Params.d

```diff
@@ -143,6 +145,10 @@ public struct ConsensusConfig
 
     /// The amount of a penalty for slashed validators
     public Amount slash_penalty_amount = 10_000.coins;
+
+    /// The minimum - transaction size adjusted - fee.
```

Comment:
All the three `-`s are minus signs?

Reply:
`- transaction size adjusted -`
here it is a dash :) I can change to ()

Reply:
Ah, `()` would be better. Ty. :)